### ISSUE TITLE
Improve parameter and method names

### ISSUE THREAD

**Issue by lassoan** [sebastianandress/Slicer-SurfaceWrapSolidify_old#2]
This effect has great potential in many applications. To give a better chance for users to successully use it, it would be great if each parameter names could be improved.

Current names are meaningful for those who very well understand the internal workings of the algorithm, but hard to grasp for users. It would be better if each parameter name reflected what the *effect* of that parameter is on the generated model: maximum distance, minimum hole size, thickness, resolution, etc. Illustrations can help a lot, where this thickness, distance, etc. values would be shown.

I could not find description of Filter modes and I'm not sure which one is for what exactly. Are "convex hull" and "raycasts" for debugging/troubleshooting only or it can be useful for some purposes with certain parameter settings? What "deep hull", and "nonmanifold" refers to? I just clicked randomly choosing modes and parameters and got very promising results, but I'm sure I could get much better results if I knew what I were doing. Similarly to the parameter names, these filter modes should reflect what the end result will be, for example instead of deep hull, non-manifold, solidified surface, it would be much more clear to use: solid, surface, shell.

**Comment by sebastianandress** [comment]
Dear @lassoan, I just wanted to let you know that I'm still onto this issue. I really like your idea with making the settings more practically oriented.

You are right, the main parameters for solidification are Convex Hull and Deep Hull; depending on your desired solidification result and origin segmentation. In fact, we added the Deep Hull method because sometimes the pure Convex Hull method failed for acetabular cups.

Especially Raycasts is just a debugging output to check the settings. We used Nonmanifold for surface comparison with deviation checking, and the final Solidified Surface was used for 3D printing fractured models.

So my idea is to just keep the Convex Hull and Deep Hull option (with better naming) and leave Solidified Surface optional (for both, convex hull and deep hull), as it is very useful for saving printing time and surely other applications. Raycast and Nonmanifold will be very hidden somewhere.

Also the Parameters will be renamed. The spacing parameters just depend on the image spacing of the origin volume/segmentation and finally the desired output. But - it increases computation time as the filter has to handle more vertices/voxels. The offset actually is just relevant for the Deep Hull Method and will be hidden somewhere in the newly ordered module.
All raycast depending parameters will be renamed like "Perforation correction factor".

Hopefully I'll get this done this month, sorry for the delay.
